mitxpro/settings.py

```diff
@@ -612,7 +615,17 @@
             600,
             description="How many seconds between Hubspot API error checks",
         ),
-    }
+    },
+    "generate-course-certificate": {
+        "task": "courses.tasks.generate_course_certificates",
+        "schedule": crontab(
+            minute=0,
+            hour=CRON_COURSE_CERTIFICATES_HOURS,
+            day_of_week=CRON_COURSE_CERTIFICATES_DAYS or "*",
```

Comment:
This shows a discrepancy between the way `CRON_COURSE_CERTIFICATES_DAYS` and `CRON_COURSE_CERTIFICATES_HOURS` is handled. One provides a default on initialization while the other provides a default on usage. They should both either set a default value on init or on usage, not sure why these are different here.

It should either be:
```
hour=CRON_COURSE_CERTIFICATES_HOURS,
day_of_week=CROUN_COURSE_CERTIFICATES_DAYS,
```

or

```
hour=CRON_COURSE_CERTIFICATES_HOURS or "0",
day_of_week=CROUN_COURSE_CERTIFICATES_DAYS or "*",
```
 with corresponding change at init.

Reply:
It is optional at that point.